db/seeds/posts/privacy-policy.html

```diff
@@ -1,65 +1,5 @@
-<h2 id="how-we-use-and-protect-your-information">How we use and protect your information</h2>

```

Comment:
We actually quite liked having a privacy policy to base ours off of, since the data collected is quite similar/the same.

The privacy policy is also linked in the footer (and required for online systems where you have some form of authentication), so I would argue that we should remove the "delete this help entry"...

Reply:
Oh, good point!  My goal was to make things easier, not harder, where "easier" includes "don't accidentally deploy terms of service (etc) that aren't what you want".

I will restore the TOS, COC, and privacy files, and take the same approach as with the spam -- leave the posts as starting points, but remove them from posts.yml so you have to turn them on (and can make edits at the same time).  Ideally we'd comment these out in the YAML file, not delete them -- assuming YAML has a comment mechanism, I'll do that.